sjb/config/common/test_cases/origin_installed_release.yml

```diff
@@ -0,0 +1,64 @@
+---
+parent: 'common/test_cases/origin_built_release.yml'
+extensions:
+  actions:
+    - type: "script"
+      title: "install the openshift-ansible release"
+      repository: "openshift-ansible"
+      script: |-
+        jobs_repo="/data/src/github.com/openshift/aos-cd-jobs/"
+        last_tag="$( git describe --tags --abbrev=0 --exact-match HEAD )"
```

Comment:
We shouldn't need to change the branch from whatever we inherit for the `build` AMI